### ISSUE TITLE
[Feature request] Last.fm scrobbler to submit both artist and album artist tags

### ISSUE THREAD

**Issue by sennowa** [DeaDBeeF-Player/deadbeef#2435]
Is there any way to get the scrobbler plugin to transmit both the main artist tag and also the album artist tag? [Here's](https://imgur.com/a/LQ5s7zj) what it looks like - the song gets added to the main album tag, but the artist is under a different tag. Right now, the plugin in deadbeef either submits just the main artist tag (and the album tag gets written to that artist alone), or, if you tick the "prefer album artist over the artist" field, it replaces the main artist tag with the album artist tag (which is not good for scrobbling compilations of different artists).

**Comment by Alexey-Yakovenko** [comment]
At the time when the last.fm plugin was developed (~10 years ago), last.fm API v1 didn't support any other metadata, than what deadbeef is transmitting. I don't think they updated it. It's possible, that last.fm API v2 supports that, but to make this supported in deadbeef, the whole plugin needs a rewrite.

**Comment by Alexey-Yakovenko** [comment]
Yeah, I can confirm that API v2 supports this, as documented here.
https://www.last.fm/api/show/track.scrobble

**Comment by Alexey-Yakovenko** [comment]
I can't find an existing feature request for implementing API v2 support, so I'll rename this one.
